blocked-edges/4.19.0-BootImageIncompatibility.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.19.0
+from: 4[.]18[.].*
+url: https://issues.redhat.com/browse/COS-3357
+name: BootImageIncompatibility
```

Comment:
maybe `OldBootImagesComposeFSvsOSProber` or something, here and in the filename, to try to pick something unique to  COS-3357, that's distinct from existing declarations like `AWSOldBootImagesLackAfterburn` and hopefully distinct from future boot-image-related regressions?

Reply:
Done.